tasks/create_homedir.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+- name: "Creating home directory for {{ item.split(':')[0] }}"
+  file:
+    path: "{{ item.split(':')[2] }}"
+    state: directory
```

Comment:
I'm not sure whether we want to create homedirs like this; usually they would at least include /etc/skel.  At least with rhel6, you wanted oddjob-mkhomedir to do it for you because of selinux issues... anyone know if this is still an issue?

Reply:
I won't claim this particular task is good for anything other than ticking off a checkbox that the home directory exists. Realistically, any normal system wouldn't have any of these to fix.